package.json

```diff
@@ -33,6 +33,7 @@
     "docs": "jsdoc2md lib/index.js > LOADER.md",
     "pretest": "npm run lint && npm run test:build",
     "test": "jest",
+    "test-only": "npm run test:build && jest",
```

Comment:
@michael-ciniawsky when i write code i want run only test to avoid regression (without lint), full test with lint and test i do before PR :smile: